content/en/blog/2024/demo-skyscanner/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title:
+  'Making observability fun: How we increased engineers confidence in incident
```

Comment:
Observability is always fun, no need to make it fun 😢 ... just kidding! great title :-)